app/src/main/java/io/github/droidkaigi/confsched/viewmodel/SessionDetailViewModel.java

```diff
@@ -78,10 +81,11 @@ public void onClickMovieIcon(@SuppressWarnings("unused") View view) {
     public void onClickFab(View fab) {
         boolean checked = !fab.isSelected();
         fab.setSelected(checked);
+
+        session = new Session(session); // create new instance for raise notifyPropertyChanged
         session.checked = checked;
         dao.updateChecked(session);
-        // TODO This is not smart way. I want to solve by using two way binding.
-        eventBus.post(new SessionSelectedChangedEvent(session));
+        observableSession.set(session);
```

Comment:
It might be simple to use `BaseObservable`. How do you think?

```java
public class SessionDetailViewModel extends BaseObservable implements ViewModel {

    @Bindable
    public Session session;
 
    public void onClickFab(View fab) {
        session.checked = !session.checked;
        notifyPropertyChanged(BR.session);
        ...
    }

}
```